src/client/editor/nodes/AmbientLightNode.js

```diff
@@ -0,0 +1,52 @@
+import * as THREE from "three";
+import EditorNodeMixin from "./EditorNodeMixin";
+import serializeColor from "../utils/serializeColor";
+
+export default class AmbientLightNode extends EditorNodeMixin(THREE.AmbientLight) {
+  static legacyComponentName = "ambient-light";
+
+  static nodeName = "Ambient Light";
```

Comment:
I think we'll probably be living with `legacyComponentName` for a while, perhaps we should just call these `serializedName` and `displayName`?

Reply:
or I guess since we use this for the GLTF component (and will for the forseeable future) perhaps just `componentName`

Reply:
I think we have a good opportunity to move to a better serialization format when we moved to a hosted version of Spoke, so this probably won't stick around long. Node name was display name before, Idk why I changed it TBH. I'll change it back.